massgov/openmass
# DP-26260 : Add field to data metadata fields so we can paste in additional content to search

**Description:**
- Added configurations for "Additional content for Data Hub search" fields
- Added dependency on the Data flag field.
- Added the new field_data_hub_search_content field to the existing hook_entity_field_access to obey access like other data fields.


**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-26260

**To Test:**
- Verify the new field called "Additional content for Data Hub search" is present on Information details, Guide, Service Details,  Curated Lists, External Link, Service Page, and Binder for Collections content types.
- Verify the "Additional content for Data Hub search field" display is hidden from the content view page.
- Verify the "Additional content for Data Hub search" field visible on the add/edit form for users with the "Access data flag fields" permission (Data Administration) role.
---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
